packages/gce-disk-expand/src/usr/share/dracut/modules.d/50expand_rootfs/expand_rootfs.sh

```diff
@@ -40,35 +40,39 @@ main() {
   disk=${out%:*}
   partnum=${out#*:}
 
-  if ! parted_needresize "$disk" "$partnum"; then
-    kmsg "Disk ${rootdev} doesn't need resizing"
-    return
-  fi
+  (
+    if ! flock -n 9; then
```

Comment:
Can you add a comment here describing why we are doing this (for future readers of this script at least).

Reply:
Also, do we need to install flock into dracut?